### PR TITLE
chore: update help text of top responses setting and translations

### PR DESCRIPTION
**Pull request by tecoholic**
**TL;DR -** Add a small change on help text of "Top Responses" setting to make it clearer to instructor. This is a re-done PR  in place of #1870 as @ngdangdat couldn't get the CLA Signed. 

JIRA: NA

**What changed?**

- Help text of the "Top Responses" setting
- Related translation files

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

* Checkout branch [open-craft:update-help-text-of-top-reponses-setting](https://github.com/open-craft/edx-ora2/tree/update-help-text-of-top-reponses-setting)
 * Install `edx-ora2` xblock on the feature branch to Studio (CMS)
 * Restart CMS
 * Add a ORA block in a unit
 * Edit > Settings
 * Make sure the changed text show up properly

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @tecoholic!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@tecoholic Thank you for your contribution, I am kicking off the tests.

**Comment by tecoholic** [comment]
@natabene Thank you.

*Note to the reviewer*: The tests are failing due to CodeCov API call failing.

**Comment by itsjeyd** [comment]
Hey @tecoholic, should we keep this PR open or can it be closed?

**Comment by tecoholic** [comment]
@itsjeyd I think it is still relevant. It is a copy of the @gabrieldamours' change [here](https://github.com/openedx/edx-ora2/compare/master...open-craft:gabriel/gabriel_opencraft_ORA_patch062422).

@gabrieldamours Can you kindly confirm?

This might require a rebase/recompiling on master. I can updated it once it is confirmed.

**Comment by itsjeyd** [comment]
Sounds good @tecoholic, thanks for the update.

**Comment by gabrieldamours** [comment]
@tecoholic yep, this PR is still relevant 👍

**Comment by itsjeyd** [comment]
@tecoholic @gabrieldamours OK great! I'll let you schedule the next steps for an upcoming OpenCraft sprint :)

@mphilbrick211 In the meantime, we should probably label this as *waiting for author* (or move it to the *Blocked* column on the contributions board)?

**Comment by tecoholic** [comment]
@itsjeyd Hi, I have re-based this on the master and updated the PR. Who should I ping to review this?

**Comment by mphilbrick211** [comment]
> 

My apologies for not seeing this. It looks like this is ready to be reviewed and merged! Looks like @mattcarter can review as he is the contact for the repo.

**Comment by itsjeyd** [comment]
@tecoholic @mphilbrick211 Thanks for the updates :+1:

**Review by jansenk — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@tecoholic 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-ora2&entry.867055334=https://github.com/openedx/edx-ora2/pull/1883&entry.1484655318=https://github.com/tecoholic&entry.752974735=2022-08-11+11:14&entry.1917517419=2022-11-10+17:31&entry.2133058324=Yes) so we can improve your experience in the future.
